src/WebJobs.Script.Scaling/IScaleHandler.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Scaling
+{
+    [SuppressMessage("Microsoft.Naming", "CA1711:IdentifiersShouldNotHaveIncorrectSuffix", Justification = "By design")]
+    public interface IScaleHandler
+    {
+        Task ScaleOut(string activityId, int workers, IWorkerInfo workerInfo);
```

Comment:
Why does that take in a workerInfo? Unclear why that's needed for scale out. Add comment?